api docker container not starting

/usr/bin/env: 'bash\r': No such file or directory
"pm ERR! Missing script: "start:prod
npm ERR! 
npm ERR! Did you mean this?
npm ERR!     npm run start:prod # run the "start:prod" package script
npm ERR! 
npm ERR! To see a list of scripts, run:
npm ERR!   npm run

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2022-07-26T17_59_15_319Z-debug-0.log

--- a/Dockerfile
+++ b/Dockerfile
@@ -9,6 +9,8 @@ COPY . /usr/src/app
 RUN cp -a /tmp/app/node_modules /usr/src/app
 COPY ./wait-for-it.sh /opt/wait-for-it.sh
 COPY ./startup.dev.sh /opt/startup.dev.sh
+RUN sed -i 's/\r//g' /opt/wait-for-it.sh
+RUN sed -i 's/\r//g' /opt/startup.dev.sh
 
 WORKDIR /usr/src/app
 RUN rm -rf .env && cp env-example .env

--- a/Dockerfile.e2e
+++ b/Dockerfile.e2e
@@ -9,6 +9,8 @@ COPY . /usr/src/app
 RUN cp -a /tmp/app/node_modules /usr/src/app
 COPY ./wait-for-it.sh /opt/wait-for-it.sh
 COPY ./startup.ci.sh /opt/startup.ci.sh
+RUN sed -i 's/\r//g' /opt/wait-for-it.sh
+RUN sed -i 's/\r//g' /opt/startup.ci.sh
 
 WORKDIR /usr/src/app
 RUN rm -rf .env && cp env-example .env

